src/core/Object3D.js

```diff
@@ -164,9 +164,10 @@ Object3D.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 
 		var q1 = new Quaternion();
 
-		return function rotateOnAxis( axis, angle ) {
+		return function rotateOnAxis( vec3, angle ) {
```

Comment:
`axis` vectors in three.js are always assumed to be unit length to prevent unnecessary normalization. We do not want to change this signature.

Reply:
use vec3 maybe simpler  ?

Reply:
>axis vectors in three.js are always assumed to be unit length

I do not support changing this convention unless it is changed uniformly throughout the library.